README.md

```diff
@@ -1,12 +1,12 @@
-# Web3.py
+# web3.py
 
 [![Documentation Status](https://readthedocs.org/projects/web3py/badge/?version=latest)](https://web3py.readthedocs.io/en/latest/?badge=latest)
 [![Discord](https://img.shields.io/discord/809793915578089484?color=blue&label=chat&logo=discord&logoColor=white)](https://discord.gg/GHryRvPB84)
 [![Build Status](https://circleci.com/gh/ethereum/web3.py.svg?style=shield)](https://circleci.com/gh/ethereum/web3.py)
 
-A Python library for interacting with Ethereum, inspired by [web3.js](https://github.com/ethereum/web3.js).
+A Python library for interacting with Ethereum.
```

Comment:
for posterity: indeed, original inspiration from web3.js (:salute:), but in the ongoing development of web3.py, that lib is no longer referenced on any regular basis, at least not any more than ethers or another language implementation.